awscli/customizations/configure/exporter.py

```diff
@@ -0,0 +1,36 @@
+from awscli.customizations.configure.writer import ConfigFileWriter
+from awscli.customizations.commands import BasicCommand
+from awscli.customizations.utils import uni_print
+import json
+
+
+class ConfigureExportCommand(BasicCommand):
+    NAME = 'export-credentials'
+    DESCRIPTION = 'Exports the AWS credentials used with the command "aws configure" to a the config file.'
+    EXAMPLES = 'aws configure export-credentials'
+
+    def __init__(self, session):
+        super(ConfigureExportCommand, self).__init__(session)
+
+    def _run_main(self, parsed_args, parsed_globals):
+        self.export_credentials(self._session)
+        return 0
+
+    def export_credentials(self, session):
+        """
+        Exports the AWS credentials used with the command 'aws configure' to a file.
+        """
+        credentials = session.get_credentials()
+        if credentials is None:
+            raise RuntimeError(
+                'No credentials available. Try running "aws configure" first.')
+        try:
+            credentials_data = credentials.get_frozen_credentials()
+            dump = {'aws_access_key_id': credentials_data.access_key,
```

Comment:
Since this is a new command, without backward compat concerns, may I suggest using the same shape as is emitted by `aws sts assume-role`? it'd be one less special case for those wishing to consume the output
```json
{
    "Credentials": {
        "AccessKeyId": "ASIA...",
        "SecretAccessKey": "sekrit...",
        "SessionToken": "bAsE64=",
        "Expiration": "2022-04-13T17:24:10.189000+00:00"
    }
}
```
where obviously `SessionToken` and `Expiration` would be elided just like they are in your current output

Reply:
Hi @mdaniel, I'll try to make the changes as suggested